Source/WebKit/UIProcess/API/Cocoa/_WKApplicationManifest.mm

```diff
@@ -364,6 +367,20 @@ - (NSString *)rawJSON
     return nullableNSString(_applicationManifest->applicationManifest().rawJSON);
 }
 
+- (_WKApplicationManifestDirection)direction
+{
+    switch (_applicationManifest->applicationManifest().dir) {
```

Comment:
Can put `using enum ApplicationManifest::Direction` at the top of the function and make the entire thing more terse without losing clarity.